test/hotspot/jtreg/vmTestbase/vm/runtime/defmeth/StressTest.java

```diff
@@ -140,23 +156,18 @@ public void run() {
     }
 
     private void configureTests() {
-        int[] majorVerValues = new int[52 - minMajorVer + 1];
+        int[] majorVerValues = new int[MAX_MAJOR_VER - MIN_MAJOR_VER + 1];
         for (int i = 0; i< majorVerValues.length; i++) {
-            majorVerValues[i] = minMajorVer + i;
+            majorVerValues[i] = MIN_MAJOR_VER + i;
         }
 
```

Comment:
Does this mean that the stress tests will run for all class versions between JDK 1.5 and JDK 17 ?  If so, maybe change it to just run for JDK 1.5, 11, and 17 releases?

Reply:
It means that the stress test will randomly pick test cases with class file versions between `MIN_MAJOR_VER` and `MAX_MAJOR_VER`. IMO it's a decent compromise between testing only  `MIN_MAJOR_VER` and `MAX_MAJOR_VER` (all other defmeth tests are configured that way) and always enumerating all the class file versions in every test (`MIN_MAJOR_VER ... MAX_MAJOR_VER`). So, I'd prefer to keep it as is.

Reply:
That sounds good.  Thanks!